lib/elixir_google_scraper/scraper/keyword.ex

```diff
@@ -19,4 +22,10 @@ defmodule ElixirGoogleScraper.Scraper.Keyword do
     |> cast(attrs, [:name, :status, :user_id])
     |> validate_required([:name, :user_id])
   end
+
+  def mark_as_completed(keyword) do
+    keyword
+    |> Keyword.changeset(%{status: :completed})
+    |> Repo.update!()
```

Comment:
The `Repo.update!()` needs to be in context. Here we will just do changeset. we can have something like

```elixir
# Here
def complete_changeset(keyword) do
  change(keyword, %{status: :completed})
end

# In context
def mark_as_completed(keyword) do
  keyword
  |> complete_changeset()
  |> Repo.update!()
end
```